### PR TITLE
[#350] Confusing validation message on bad indentation

### PR DESCRIPTION
**Pull request by ghillairet**
See #350

### CODE REVIEW

File: com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/Activator.java [hunk 1]
```diff
@@ -97,7 +95,7 @@ public static Activator getDefault() {
 	}
 	
 	public void logError(String message, Exception e) {
-		getLog().log(new Status(IStatus.ERROR, PLUGIN_ID, message, e));
+        // getLog().log(new Status(IStatus.ERROR, PLUGIN_ID, message, e));
```
Comment (tfesenko) [inline com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/Activator.java]: Is it intentional? If we call `logError` we probably want this error logged :)

File: com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/editor/JsonDocument.java [hunk 2]
```diff
@@ -162,10 +162,10 @@ private void parseJson(String content) {
     }
 
     private void parseModel() {
+        System.out.println("parse...");
```
Comment (tfesenko) [inline com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/editor/JsonDocument.java]: Leftover sysout?

File: com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/Validator.java [hunk 3]
```diff
@@ -106,6 +106,7 @@ public Validator(JsonReferenceValidator referenceValidator, Map<String, JsonNode
         try {
             jsonContent = document.asJson();
         } catch (Exception e) {
+            System.out.println("Loggg");
```
Comment (tfesenko) [inline com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/Validator.java]: The same here, people often forget to remove debugging logs

File: com.reprezen.swagedit.tests/src/com/reprezen/swagedit/validation/YamlErrorProcessorTest.xtend [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+			    Pet:
+			      required:
+			        - id
+			      properties:
+			        id:
+			          type: integer
+			          format: int64
+			       name:
+			         type: string
+		'''
```
Comment (tfesenko) [inline com.reprezen.swagedit.tests/src/com/reprezen/swagedit/validation/YamlErrorProcessorTest.xtend:33]: Thanks!

File: com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/YamlErrorProcessor.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        return exception != null ? exception.getLocalizedMessage() : null;
+    }
+
+    public String rewriteMessage(YAMLException exception) {
+        if (exception instanceof ParserException) {
+            if ("while parsing a block mapping".equals(((ParserException) exception).getContext())) {
+                return Messages.error_yaml_parser_indentation;
+            }
+        }
+        return exception != null ? exception.getLocalizedMessage() : null;
```
Comment (tfesenko) [inline com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/YamlErrorProcessor.java:36]: Nice

**Comment by tfesenko** [comment]
Passed QA
Current message:
> Unable to parse content, an incorrect indentation may be the cause.
screen shot 2017-06-09 at 10 34 40 am

**Comment by tfesenko** [comment]
Passed code review, but suggested to remove two residual sysouts and restore the `logError()`.

File: com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/Validator.java [hunk 6]
```diff
@@ -106,7 +106,6 @@ public Validator(JsonReferenceValidator referenceValidator, Map<String, JsonNode
         try {
             jsonContent = document.asJson();
         } catch (Exception e) {
-            System.out.println("Loggg");
```
Comment (tfesenko) [inline com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/validation/Validator.java]: Thanks!

**Comment by ghillairet** [comment]
I removed the output logs.

**Pull request merged**